core/vm/actor/builtin/types/market/policy.hpp

```diff
@@ -25,7 +25,7 @@ namespace fc::vm::actor::builtin::types::market {
                                           * TokenAmount{"1000000000000000000"}};
 
   /// The number of blocks between payouts for deals
-  constexpr EpochDuration kDealUpdatesInterval{fc::kEpochsInDay};
+  static EpochDuration kDealUpdatesInterval = fc::kEpochsInDay;
```

Comment:
Why static? constexpr looks better

Reply:
It is not `const` anymore, changed to extern.